src/WidenImage.css

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
:+1: sounds good! BTW, I believe this CSS uncovers a [bug in the CSS validator](https://github.com/w3c/css-validator/issues/72).